src/answers-headless.ts

```diff
@@ -60,6 +59,14 @@ export default class AnswersHeadless {
     this.stateManager.dispatchEvent('universal/setLimit', limit);
   }
 
+  setFilterSearchQuery(query: string, filterSearchId: string): void {
```

Comment:
I'm wondering if there's value in storing the filterSearchQuery in state actually. The normal search query, I think makes sense to store. There are a number of consumers of it and applications for it. I get the feeling that anyone setting a filterSearch query will want to fire a filterSearch almost immediately after. 

What would you think about removing this from the state for now and have it be passed into `executeFilterSearch`?